### ISSUE TITLE
ftp cannot send files after mfdata update

### ISSUE THREAD

**Issue by matthieumarrast** [metwork-framework/mfdata#456]
Rights to folder `/home/mfdata` are set to null (0) for "other" users

```
Password:
500 OOPS: cannot change directory:/home/mfdata/var/in/incoming
Login failed.
421 Service not available, remote server has closed connection
```

Bypass - after each mfdata update:
`chmod o+x /home/mfdata`

**Comment by thebaptiste** [comment]
Other bypass : at admin level, add once for all "ftp" user(s) in metwork group.

We will probably change nothing on the subject in Metwork, for the time being at least.

**Comment by thebaptiste** [comment]
Closed by #1525
